setup.cfg

```diff
@@ -48,6 +48,7 @@ install_requires =
     sentencepiece>=0.1.99
     peft@git+https://github.com/huggingface/peft@5884bdbea49e5e71e2cd06ecfa484bb635063735
     safetensors>=0.3.1
+    Dijkstar>=2.6.0
```

Comment:
nit: perhaps we should set <3.0.0 to protect against interface changes

Reply:
Here, I use only the simplest graph interface - I think it's unlikely to be changed and/or can be quickly updated.